pkg/processor/procbuiltin/httprequest.go

```diff
@@ -33,6 +34,8 @@ const (
 
 	httpRequestConfigURL          = "url"
 	httpRequestConfigMethod       = "method"
+	httpRequestConfigContentType  = "contentType"
+	httpRequestContentType        = "Content-Type"
```

Comment:
Nitpick: if you ask me, I'd write the header string directly in code, imo it would make it easier to read. Then again, the `application/json` default content type could be a constant, since you wouldn't have to start reading the processor to figure out what the default is.